projects/desktop/src/components/common/SingleForm/checkbox/CheckboxInput.tsx

```diff
@@ -1,20 +1,21 @@
 import styled from 'styled-components'
-import {colors} from '../../../../constants'
+
+import { colors } from '../../../../constants'
 
 export const CheckboxInput = styled.input`
     position: absolute;
     opacity: 0;
     cursor: pointer;
     height: 0;
     width: 0;
-    
+
     :checked ~ span {
```

Comment:
Can you remember me this selector?

Reply:
![Annotation 2019-11-30 093427](https://user-images.githubusercontent.com/9156538/69898076-a08d0b00-1354-11ea-9425-3b44c13c6b61.png)